packages/twenty-front/src/modules/page-layout/widgets/widget-card/components/WidgetCard.tsx

```diff
@@ -0,0 +1,163 @@
+import { isPageLayoutInEditModeComponentState } from '@/page-layout/states/isPageLayoutInEditModeComponentState';
+import { useRecoilComponentValue } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValue';
+import styled from '@emotion/styled';
+import { type ReactNode } from 'react';
+import { isDefined } from 'twenty-shared/utils';
+
+import { type WidgetCardContext } from '../types/WidgetCardContext';
+
+export type WidgetCardProps = {
+  children?: ReactNode;
+  widgetCardContext?: WidgetCardContext;
+  onClick?: () => void;
+  isEditing?: boolean;
+  isDragging?: boolean;
+  className?: string;
+};
+
+const StyledWidgetCard = styled.div<{
+  onClick?: () => void;
+  widgetCardContext?: WidgetCardContext;
```

Comment:
I think `widgetCardContext` shouldn't be optional in the component, as you provide a default value to the component's prop. Furthermore, it doesn't make much sense to me to render the component without context now.